django/db/backends/sqlite3/_functions.py

```diff
@@ -36,6 +35,11 @@
 from django.utils.crypto import md5
 from django.utils.duration import duration_microseconds
 
+try:
+    import zoneinfo
+except ImportError:
+    from backports import zoneinfo
+
```

Comment:
`zoneinfo` is universally available from Python 3.9. 

5.0 is 3.10+ no? 

Can we just import it straight?

Reply:
There will be days not too far ahead with Python 3.10+ only!? Can't wait! 😍

Reply:
Yes we can, and we will change it later as a part of ticket-34233.